.editorconfig

```diff
@@ -1,16 +0,0 @@
-root = true
-
-[*]
-charset = utf-8
-end_of_line = lf
-insert_final_newline = true
-trim_trailing_whitespace = true
-indent_size = 2
-indent_style = space
-
-[*.md]
-trim_trailing_whitespace = false
-
-[*.py]
-indent_size = 4
-indent_style = space
```

Comment:
Seems in most editors this file is only usable with an additional plugin anyway. Not sure anyone actually uses this? I don't.